components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -278,7 +291,7 @@ subroutine linv3_strat_chem_solve(ncol, lchnk, xvmr, h2ovmr, xsfc, o3col,temp, s
     real(r8), intent(in)                           :: delta_t             ! timestep size (secs)
     real(r8), intent(in)                           :: rlats(ncol)         ! column latitudes (radians)
     integer,  intent(in)   , dimension(pcols)      :: ltrop               ! chunk index    
-    real(r8), intent(in)   , dimension(pcols,pver) :: pdeldry             !  dry pressure delta about midpoints (Pa) 
+    real(r8), intent(in)   , dimension(pcols ,pver) :: pdeldry             !  dry pressure delta about midpoints (Pa) 
```

Comment:
A small change. Remove the space after 'pcols', .i.e., reverting the changes seen by git.